storage/storage_tests/storage_tests.cpp

```diff
@@ -1227,4 +1221,39 @@ UNIT_TEST(StorageTest_ParseStatus)
   TEST_EQUAL(StatusAndError(NodeStatus::Downloading, NodeErrorCode::NoError),
              ParseStatus(Status::EDownloading), ());
 }
+
+UNIT_TEST(StorageTest_ForEachInSubtree)
+{
+  Storage storage(kSingleMwmCountriesTxt, make_unique<TestMapFilesDownloader>());
+
+  TCountriesVec leafVec;
+  auto const forEach = [&leafVec](TCountryId const & descendantCountryId, bool expandableNode)
```

Comment:
descendantCountryId contains it's type in it's name. When you write "CountryId const & descendantCountryId" it's the same as "int intN = 10; double doubleF = 20.0; ..."